tests/acceptance/features/webUIFilesDetails/fileDetails.feature

```diff
@@ -18,9 +18,9 @@ Feature: User can open the details panel for any file or folder
     Then the app-sidebar should be visible
     And the thumbnail should be visible in the app-sidebar
     And the "details" details panel should be visible
-    When the user switches to "actions" accordion item in details panel using the webUI
+    When the user switches to "actions" panel item in details panel using the webUI
```

Comment:
`panel item` looks redundant to me. Could you refactor to just `panel` instead of `panel item`?